OCP/AdaptGuess_UserInput.m

```diff
@@ -28,7 +28,10 @@
 % Last edit date: 
 % --------------------------------------------------------------------------
 
-guess.a(guess.a < S.bounds.a.lower) = S.bounds.a.lower;
+for i=1:length(bounds.a.lower)
+    guess.a(guess.a(:,i) < bounds.a.lower(i),i) = bounds.a.lower(i);
+end
+
```

Comment:
For my understanding, why only do this for the activations? Wouldn't it be possible that your IG is also out of bounds when basing the bounds on a suggested Default_Coordinate_Bounds.csv file and not the IG itself like before?

Reply:
ipopt will push the initial guess to fit between the bounds (https://coin-or.github.io/Ipopt/OPTIONS.html#OPT_bound_push), so we don't really need to care about this.
I removed this function.